src/Zendesk/Types.hs

```diff
@@ -177,6 +224,29 @@ data TicketInfo = TicketInfo
     , ticketStatus :: !Text        -- ^ The status of the ticket
     } deriving (Eq, Show)
 
+
+-- TODO(ks): Newtype!
+instance Arbitrary TicketURL where
+    arbitrary = do
+        protocol    <- elements ["http://"]
+        name        <- listOf1 $ elements ['a'..'z']
+        domain      <- elements [".com",".com.br",".net"]
+        pure . TicketURL . pack $ protocol ++ name ++ domain
+
+instance Arbitrary TicketInfo where
+    arbitrary = do
+        ticketId     <- arbitrary
+        ticketUrl    <- arbitrary
+        ticketTags   <- pure []
+        ticketStatus <- pure "open"
+
+        pure TicketInfo
+            { ticketId      = ticketId
+            , ticketUrl     = ticketUrl
+            , ticketTags    = ticketTags
+            , ticketStatus  = ticketStatus
+            }
```

Comment:
This can be written much simpler like this `pure TicketInfo{..}`

Reply:
Which is not explicit. While I will switch from one style to the other, depending on the number of parameters, this is the correct way to do it and to be as explicit as possible.